synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -122,6 +125,14 @@ ffmpeg_trgt::~ffmpeg_trgt()
 	file=NULL;
 	delete [] buffer;
 	delete [] color_buffer;
+
+	// Remove temporary sound file
+	struct stat buf;
```

Comment:
Use GStatBuf: https://developer.gnome.org/glib/stable/glib-File-Utilities.html#g-stat

Reply:
Got it! Thank you for the hint. ^__^